docs/dev/facets.rst

```diff
@@ -0,0 +1,54 @@
+Facets
+======
+
+Facets serve as a grouping of different classes.
+Within the data models used in OS2MO, model fields corresponding to a facet
+name is expected to contain a reference to a class contained in that facet.
+E.g. the field ``responsibility`` in the ``manager`` data model, contains a
+reference to a class contained in the ``responsibility`` facet.
```

Comment:
Mention that these are proper OIO objects? And that we disregard the actual classification?